docs/xls-65d-single-asset-vault/concepts/single-asset-vault.md

```diff
@@ -29,7 +29,7 @@ Choosing between a public or private vault depends on your use case. For example
 
 Depositors can deposit assets to receive shares, which represent their proportional ownership of the vault, or redeem shares for assets.
 
-[{% inline-svg file="./single-asset-vault.svg" /%}](./single-asset-vault.svg "Diagram: an example of an asset being deposited into the vault and shares being redeemed.")
+[{% inline-svg file="./single-asset-vault-img.svg" /%}](./single-asset-vault-img.svg "Diagram: an example of an asset being deposited into the vault and shares being redeemed.")
```

Comment:
This might seem odd, but I noticed some weird behaviour in which Redocly was appending a "-1" at the end of the page url.

See: https://opensource.ripple.com/docs/xls-65d-single-asset-vault/concepts/single-asset-vault-1

Turns out I had named the image the exact same name as the page, which seems to cause this behaviour for some reason. So I changed the image name for that reason.